BridgeSDKTests/MockURLSession.m

```diff
@@ -84,14 +84,14 @@ - (void)resume
 {
     [_session.delegateQueue addOperationWithBlock:^{
         MockHTTPURLResponse *response;
-        __unused NSData *data = [_session dataAndResponse:&response forRequest:_request];
-        _mockResponse = response;
+        __unused NSData *data = [self->_session dataAndResponse:&response forRequest:self->_request];
```

Comment:
Was this set of changes to fix warnings or a crash? Wouldn't capturing self and dereferencing in the block, instead of capturing the individual _session, _request, or _mockResponse as they existed at the time of the call, lead to unexpected behavior if the values happen to get changed between when the block was added to the delegate queue and when it got executed? Not that that would ever happen in typical use, just making sure there's a method behind this "madness"...

Reply:
Fix a warning. When you are using them in blocks, should use self to explicitly retain rather than implicitly retain (because you are referencing an ivar).